teamA/sangyeop/db/migrations/20220804020745_create_userID_table.sql

```diff
@@ -0,0 +1,13 @@
+-- migrate:up
+CREATE TABLE userId (
```

Comment:
테이블 명이 userId로 되어 있네요. users로 수정해주세요.

```suggestion
CREATE TABLE users (
```